lib/getContractSources.js

```diff
@@ -5,21 +5,46 @@ import path from 'path'
  * Extract contents of contract files as string
  *
  * @param  {Array}  contracts             Contracts to compile
+ * @param  {String} modulesPath           Path to node_modules directory
  * @param  {String} inputPath             Directory to read target contracts from
  * @param  {String} contractFileExtension File extension of the target contracts
  * @return {Array}                        Promises resolving contract file content
  */
-const getContractSources = (contracts, inputPath, contractFileExtension) => {
+const getContractSources = (
+  contracts,
+  modulesPath,
+  inputPath,
+  contractFileExtension
+) => {
   return contracts.map(async contract => {
     try {
+      // redeclare injections to allow for them to overriden
+      let filePath = inputPath
+      let contractFilename = contract
+
+      // Check for contract dependencies installed via npm
+      if (contractFilename.includes('/')) {
```

Comment:
This should be improved to be `startsWith`, to allow for directory structure within target directory.